src/utils/api.js

```diff
@@ -1,5 +1,5 @@
-function getArticles(){
-    return fetch(`https://dom-nc-news-web.onrender.com/api/articles`)
+function getArticles(searchquery = "articles"){
```

Comment:
this request will always just need to get the articles so you don't need the search query yet - can just have the string as /api/articles without expecting there to be an argument at this point in time (as this function is getting the articles, it's always going to have the /articles part in the url so this doesnt need to be dynamic)